### ISSUE TITLE
SSH access to server

### ISSUE THREAD

**Issue by Reasat** [CancerDataScience/HistomicsML-TA#65]
I am trying to run the histomicsML docker container on a server and access it via ssh from my local computer. I am not sure which port to ssh in.

From https://histomicsml2.readthedocs.io/en/latest/data-import.html
After I execute hml container 
`docker run --net hmlnet -i -t -v "$PWD":/datasets -p 80:80 -p 6379:6379 --link hmldb --name hml cancerdatascience/histomicsml:1.0 /bin/bash
`and launch HistomicsML 
`python run_model_server.py
`
The instruction is to browse to http://localhost/HistomicsML/data.html

But I need to ssh from my local computer to the port in the server where the HistomicsML interface is running.

So how can I figure out the port address?

There seem to be three containers, hml, hmldb. and hmlnet. I am also confused about which container I should ssh into. _Since hmlnet is a server for the database containers (_hml and hmldb?_) to communicate over_, should I ssh into hml?

**Comment by slee172** [comment]
@Reasat "hmlnet" is not a docker container but a docker network. hml is a server container and hmldb is a database container. You can see more details here https://histomicsml2.readthedocs.io/en/latest/system-overview.html
Just for your information, you can set the port using the "-p" option. For example, -p 80:80 is the port for the apache server.

**Comment by Reasat** [comment]
Thanks!

I changed the port in the example instruction 
(https://histomicsml2.readthedocs.io/en/latest/example-data.html)
from 80 to 8880 using and ran it in my server
`docker run --net hmlnet -i -t -p 8880:80 -p 6379:6379 --link hmldb --name hml cancerdatascience/histomicsml:example /bin/bash`

Then I sshed into port 8880 from my local computer.

I can access the histomicsML interface from localhost:8880\HistomicsML.

But after I select the BRCA data and start a session, the image does not load

![image](https://user-images.githubusercontent.com/15989033/98700647-085a1d00-233e-11eb-85ad-f61ee856c9e9.png)

Any advice on where to look to solve this?
